impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -221,12 +221,17 @@ Scalar GaussianBlurFilterContents::CalculateScale(Scalar sigma) {
   exponent = std::max(-4.0f, exponent);
   Scalar rounded = powf(2.0f, exponent);
   Scalar result = rounded;
-  // Only drop below 1/8 if 1/8 would overflow our kernel.
+  // Extend the range of the 1/8th downsample based on the effective kernel size
+  // for the blur.
   if (rounded < 0.125f) {
     Scalar rounded_plus = powf(2.0f, exponent + 1);
     Scalar blur_radius = CalculateBlurRadius(sigma);
     int kernel_size_plus = (ScaleBlurRadius(blur_radius, rounded_plus) * 2) + 1;
-    result = kernel_size_plus < kMaxKernelSize ? rounded_plus : rounded;
+    // This constant was picked by looking at the results in to make sure no
+    // shimmering was introduced.
```

Comment:
Looks like there's a missing word or words in this comment in "looking at the results in to make sure...".

Reply:
done, fixed and expanded